dsl/camel-endpointdsl/pom.xml

```diff
@@ -44,6 +44,11 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-core-engine</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.sun.mail</groupId>
+            <artifactId>jakarta.mail</artifactId>
```

Comment:
Looks weird, I don't think it should be here.

Reply:
was failing for me in a full local build. that's why i had to add it. i can remove it if it is only for me.

Reply:
Yeah lets remove it, as CI builds are okay, and we have migrated to JakartaEE and Angular Mail 2